tabs/offering.md

```diff
@@ -76,7 +68,6 @@ lng_pair: id_offering
                         <li>Strength, empathy, respect and kindness (Interpersonal Skills)</li>
                     </ul>
                 </div>
+                </div>
```

Comment:
The indentation of the file is wrong. You can check the formatting options in your IDE to fix it automatically.